ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,467 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+
+import rdflib
+from rdflib import URIRef
+from rdflib import Graph
+
+
+class G2PDataset:
+    """
+    an rdf object store
+    """
+
+    # keep the graph as a static variable
+    g = None
+    # overwrite tests/unit/cgd-test.ttl
+    record_test = False
+    #
+    g2 = Graph()
+
+    def __init__(self, is_prod):
+
+        self.search_query = """
```

Comment:
We tend to make instance variables protected by default. So, this should be ``self._search_query``.